homeassistant/components/fibaro/manifest.json

```diff
@@ -5,5 +5,6 @@
   "requirements": ["fiblary3==0.1.8"],
   "codeowners": [],
```

Comment:
You're doing a lot of good work here.  Sure you don't want to be codeowner ?

Reply:
I added myself as codeowner.